Lib/importlib/_bootstrap_external.py

```diff
@@ -211,10 +212,10 @@ def _write_atomic(path, data, mode=0o666):
         with _io.FileIO(fd, 'wb') as file:
             file.write(data)
         _os.replace(path_tmp, path)
-    except OSError:
+    except (OSError, ValueError):
         try:
             _os.unlink(path_tmp)
-        except OSError:
+        except (OSError, ValueError):
```

Comment:
How can it raise ValueError?

Reply:
During the `write` it shouldn't actually. My bad!